tests/functional/tests.py

```diff
@@ -1790,6 +1853,12 @@ def main():
                 shutil.copyfileobj(LimitedRandomReader(1024*1024), file_data)
             with open(largefile, 'wb') as file_data:
                 shutil.copyfileobj(LimitedRandomReader(11*1024*1024), file_data)
+            with open( csvfile, 'w') as file_data:
```

Comment:
I am not sure if the editor you are using has issues, looks like formatting is not working correctly - you may want to fix that on your end to avoid these type of formatting issues.
```suggestion
            with open(csvfile, 'w') as file_data:
```

Reply:
done